### ISSUE TITLE
Why detectron2 don't detect text object?

### ISSUE THREAD

**Issue by HamdiTarek** [facebookresearch/detectron2#3611]
I am using detectron2 to detect brands in images? 
Number of classes: 100
Number of images/class: about 3000
Train size: 280k images
Time for one epoch: About 4 days
I tried to resize images to 512*512 using  T.Resize((512, 512)) in the data augmentation, then I found out that there is # Maximum size of the side of the image during training  **_C.INPUT.MAX_SIZE_TRAIN** =  new_size (512 I'm my case). Based on that, did the first epoch use the original image size?

Configuration: 
def cfg_setup():
    # Load default config
    cfg = get_cfg()
    cfg.MODEL.DEVICE=DEVICE

    # merge with specific config chosen before
    cfg.merge_from_file(model_zoo.get_config_file(MODEL_PATH))
    # since internet is not available specify exact path to the weights
    cfg.MODEL.WEIGHTS = WEIGHT_PATH
    # specify number of classes
    cfg.MODEL.RETINANET.NUM_CLASSES = nc
    cfg.MODEL.ROI_HEADS.NUM_CLASSES = nc
    
    
    # Number of regions per image used to train RPN
    cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 512

    
    # set trianing dataset
    cfg.DATASETS.TRAIN = ('brand_train',)
    cfg.DATASETS.TEST = ()
    
    # Number of data loading threads
    cfg.DATALOADER.NUM_WORKERS = 8
    
    cfg.SOLVER.IMS_PER_BATCH = 32
    
    # Learning rate scheduler
    cfg.SOLVER.LR_SCHEDULER_NAME = 'WarmupCosineLR'
    cfg.SOLVER.BASE_LS = 0.0002
    cfg.SOLVER.MAX_ITER = 10000000

    num_gpu = 1
    bs = (num_gpu * 2)
    cfg.SOLVER.BASE_LR = 0.02 * bs / 16  # pick a good LR
    # create a folder for the output files
    cfg.OUTPUT_DIR=OUTPUT_DIR
    os.makedirs(OUTPUT_DIR, exist_ok=True)
    
    return cfg

**After one epoch the model can detect many brands logos, but he completely fails with brand text logo (A brand based on text, characters)**
The annotated images sometimes they are missing some boxes but after training my model can detect other boxes: For example, some images have 4 bboxes for the same object (let's say a star) but there are 7 stars in the images, in the testing my model can detect these 3 missing starts in the annotations. What are the properties evaluation metrics that I need to use in this case?
 **Example: in the left ground truth and in the right is the predictions:**

![Capture](https://user-images.githubusercontent.com/19616128/138510608-40937be0-f8fd-4961-8d9c-4a4daca0ce16.JPG)

I tried to test current epoch: I used **COCOEvaluator** and this time I fixed the size of the image inside the config function:
def cfg_test():
    cfg = get_cfg()
    cfg.merge_from_file(model_zoo.get_config_file(MODEL_PATH))
    cfg.MODEL.WEIGHTS = 'detectron2/output_V2/model_0154999.pth'
    cfg.DATASETS.TEST = ('brand_test',)
    # Size of the smallest side of the image during testing. Set to zero to disable resize in testing.
    cfg.INPUT.MIN_SIZE_TEST = 512
    # Maximum size of the side of the image during testing
    cfg.INPUT.MAX_SIZE_TEST = 512
    cfg.MODEL.RETINANET.NUM_CLASSES = nc
    cfg.MODEL.ROI_HEADS.NUM_CLASSES = nc
    #cfg.MODEL.RETINANET.SCORE_THRESH_TEST = 0.6
    cfg.MODEL.ROI_HEADS.SCORE_THRESH_TEST = 0.1  # set a testing threshold
    #predictor = DefaultPredictor(cfg)
    
##############################################

In the Custum dataset creation for training and testing I used:
           "bbox": (
                        x,
                        y,
                        x + w,
                        y + h,
                        ),
           "bbox_mode": BoxMode.XYXY_ABS,
Did the **COCOEvaluator**  use the same BoxMode.XYXY_ABS

I have many questions concerning detection2 and object detection. I hope I will get answers ASAP.
Thank you all.

**Comment by ppwwyyxx** [comment]
As mentioned above:

> If you expect a model to converge / work better, we do not help with such issues, unless a model fails to reproduce the results in detectron2 model zoo, or proves existence of bugs.

We provide configs & models with standard academic settings and expect users to have the knowledge to choose or design appropriate models & parameters for their own tasks.
